reflectapi-schema/src/lib.rs

```diff
@@ -18,6 +21,8 @@ pub struct Schema {
     pub output_types: Typespace,
 }
 
+type Subst = HashMap<String, TypeReference>;
```

Comment:
Is it used in many places? can we use the HashMap<> directly there... it was very difficult to read the code where it is used.